tests/install_testbed/install_neutron_controller.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash -ex
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FIWARE project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+# Authors: Jose Ignacio Carretero Guarde and Chema
+
+. config_vars
+
+# create the database
+./create_mysqldb $NEUTRON_DB $NEUTRON_DBPASSWORD
+
+
+### install packages
+apt-get install -y neutron-server neutron-plugin-ml2 python-neutronclient
+
+### configure
+file=/etc/neutron/neutron.conf
+./openstack-config --set $file database connection mysql://$NEUTRON_DBUSER:$NEUTRON_DBPASSWORD@$CONTROLLER/$NEUTRON_DB
+./openstack-config --set $file DEFAULT verbose True
+./openstack-config --set $file DEFAULT rpc_backend rabbit
+./openstack-config --set $file DEFAULT rabbit_host $CONTROLLER
+./openstack-config --set $file DEFAULT rabbit_password $RABBIT_PASS
+./openstack-config --set $file DEFAULT auth_strategy keystone
+./openstack-config --set $file keystone_authtoken auth_uri $AUTH_URI
+./openstack-config --set $file keystone_authtoken identity_uri $IDENTITY_URI
+./openstack-config --set $file keystone_authtoken admin_tenant_name service
+./openstack-config --set $file keystone_authtoken admin_user neutron
+./openstack-config --set $file keystone_authtoken admin_password $NEUTRON_PASS
+./openstack-config --set $file DEFAULT core_plugin ml2
+./openstack-config --set $file DEFAULT service_plugins router
+./openstack-config --set $file DEFAULT allow_overlapping_ips True
```

Comment:
This configuration, althought is only for testing is not the same in the FIWARE Lab, at least in Spain, if I could remember we do not allow the overlapping of IPs... Just FYI